src/plugins/discover/public/application/main/components/top_nav/discover_topnav.tsx

```diff
@@ -39,7 +40,10 @@ export const DiscoverTopNav = ({
   resetSavedSearch,
 }: DiscoverTopNavProps) => {
   const history = useHistory();
-  const showDatePicker = useMemo(() => indexPattern.isTimeBased(), [indexPattern]);
+  const showDatePicker = useMemo(
+    () => indexPattern.isTimeBased() && indexPattern.type !== DataViewType.ROLLUP,
```

Comment:
:information_source: This now hides the time picker also for rollup index patterns.